android/scripts/release-sdk.sh

```diff
@@ -38,17 +38,19 @@ if [[ $MVN_HTTP == 1 ]]; then
         -DgeneratePom=false \
         -DpomFile=react-native-${RN_VERSION}.pom || true
     popd
-    # Push JSC
-    echo "Pushing JSC ${JSC_VERSION} to the Maven repo"
-    pushd ${THIS_DIR}/../../node_modules/jsc-android/dist/org/webkit/android-jsc/${JSC_VERSION}
+    # Push Hermes
+    echo "Pushing Hermes ${HERMES_VERSION} to the Maven repo"
+    pushd ${THIS_DIR}/../../node_modules/hermes-engine/android/
     mvn \
         deploy:deploy-file \
         -Durl=${MVN_REPO} \
         -DrepositoryId=${MVN_REPO_ID} \
-        -Dfile=android-jsc-${JSC_VERSION}.aar \
+        -Dfile=hermes-release.aar \
```

Comment:
You don't want the file name to be versioned just like JSC was?

Reply:
The file is versioned. When it reaches Maven it's called "hermes-$version.aar" this path refers to the local copy of the file you're publishing.